docs/_utilities/color.md

```diff
@@ -12,7 +12,7 @@ The color codes listed beneath each name indicate what names can be used with th
   This element has classes <code>bg-primary-lighter</code> and <code>text-primary-dark</code>.
 </div>
 
-Read more about <a href="https://v2.designsystem.digital.gov/utilities/color/" target="_blank">the U.S. Web Design System’s color utility classes</a>.
+Read more about [the U.S. Web Design System’s color utility classes](https://designsystem.digital.gov/utilities/color/).
```

Comment:
removing `target="_blank"` is intentional right? the kramdown doesn't add it back in

Reply:
> removing `target="_blank"` is intentional right? the kramdown doesn't add it back in

Good spot and yes, `target="_blank"` will be removed with these changes. I don't think this page qualifies for any of the standard exceptions of [opening a new tab only when necessary](https://www.w3.org/TR/WCAG20-TECHS/G200.html), so I didn't care much to retain it.